README.md

```diff
@@ -1,7 +1,50 @@
 # pyCascadia
 Implementation of GEBCO cookbook remove-restore and other cleaning of topography/bathymetry. Uses `pyGMT`.
 
-## Shape file closing
+## Installation
+Some dependencies of the package are a bit fiddly to install, especially on Windows.
+We recommend to use a conda environment, and install the more difficult dependencies through `conda-forge`,
+by running the commands below in sequence:
+1. Clone this GitHub repository.
+```
+git clone https://github.com/UCL/pyCascadia.git
+```
+2. In your local copy of the GitHub repository, create a conda environment (named "cascadia" here, but you might want to choose a different name) with Python 3.9. Other Python versions might also work, but the code is only actively tested with Python 3.9. This step also installs `pip`, which will be used to install the more straight-forward dependencies.
+```
+conda create -n cascadia python=3.9 pip
+```
+3. Additionally, install `pygmt`, `gdal` and `rasterio` into the "cascadia" environment via `conda-forge`. This step is needed because these packages are not available from the Python Package index (PyPi) (which is where `pip` looks by default) for all operating systems.
+```
+conda install -n cascadia pygmt gdal rasterio -c conda-forge
+```
+4. Activate the "cascadia" environment
+```
+conda activate cascadia
+```
+5. Install `pyCascadia`. This will also automatically install the remaining dependencies `matplotlib`, `pytest`, and `xarray`.
```

Comment:
The dependencies are already explicitly listed in `requirements.txt` so listing them here feels like unnecessary duplication (which will become uncoupled when those dependencies change). 

```suggestion
5. Install `pyCascadia`. This will also automatically install the remaining dependencies.
```

Reply:
Yes, good point. My reasoning for including them here was that novice users (that don't know about `requirements.txt`) know what they're installing/need to have installed (which might help with track down causes of any installation issues encountered). Maybe the best of both worlds is to link to `requirements.txt` from here?

Reply:
Actually, seems like good practice for packages now is listing requirements in the setup.cfg (see [pandas setup.cfg](https://github.com/pandas-dev/pandas/blob/6ea277f2417f4454c1762cbad472468663007f28/setup.cfg#L33)). We should probably just do that and simplify installation further.